runtime/ddr/run_omrddrgen.mk.ftl

```diff
@@ -27,10 +27,8 @@ include $(TOP_DIR)makelib/uma_macros.mk
 
 ifeq (8,$(VERSION_MAJOR))
   DDR_JCL_MODULE := jclse7b_
-else ifneq (,$(filter 9 10 11,$(VERSION_MAJOR)))
-  DDR_JCL_MODULE := jclse$(VERSION_MAJOR)_
 else
-  $(error Unsupported version: '$(VERSION_MAJOR)')
+  DDR_JCL_MODULE := jclse$(VERSION_MAJOR)_
```

Comment:
This may not work in the future, but I suppose it can be fixed when it breaks. We only made 9, 10, 11 versions because something changed which required a new version. This may not always be the case.